src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -352,16 +385,14 @@ const AppointmentDetails = props => {
                   </div>
                 )}
               </div>
-              {!isUpcoming &&
-                eligibleAppointment && (
-                  <div className="vads-u-margin-top--2">
-                    <ActionLink
-                      app={app}
-                      action={action}
-                      appointmentId={getAppointmentId(appointment)}
-                    />
-                  </div>
-                )}
+              {app === APP_NAMES.CHECK_IN && link}
+              {app === APP_NAMES.PRE_CHECK_IN && (
```

Comment:
This is technically covered but there's no assertion checking for it on pre check in.

Reply:
Added checks for both day-of and pre-check-in